app/src/main/java/com/keyboardbuilder/dto/KeyboardCase.java

```diff
@@ -0,0 +1,39 @@
+package com.keyboardbuilder.dto;
+
+public class KeyboardCase
+{
+    private String material;
+    private int layout;
+
+    public KeyboardCase()
+    {
+        this.material = "Test";
+        this.layout = 1;
+    }
+
+    public KeyboardCase(String material, int layout)
+    {
+        this.material = material;
+        this.layout = layout;
+    }
+
+    public String getMaterial()
+    {
+        return material;
+    }
+
+    public void setMaterial(String material)
+    {
+        this.material = material;
+    }
+
+    public int getLayout()
+    {
+        return layout;
+    }
+
+    public void setLayout(int layout)
+    {
+        this.layout = layout;
+    }
```

Comment:
A good chunk of these methods can be replaces with Lambok functions learned in class. Due to the fact that there's no project set up for an IDE to work off of, this "plain" Java is the best I could do. @Getter and @Setter functions from Lambok should be easy to replace majority of these functions.